ulmo/ssl/resnet_big.py

```diff
@@ -0,0 +1,209 @@
+"""ResNet in PyTorch.
```

Comment:
Consider whether this should be in this folder or in the more generic `models` folder.